ffv1.md

```diff
@@ -710,6 +733,8 @@ Figure: A pseudo-code description of the read of a signed integer in Golomb Rice
 |0000 0000 0001 | 11    |
 |0000 0000 0000 | ESC   |
 
+`ESC` (meaning "Escape code") case is choosen when it is considered more efficient to store the whole number compared to storing the least significant bits of the number.
```

Comment:
`ESC` is already defined in Section 2.1. Is this a redefinition? If it is defined here, could it be removed from Section 2.1 so there are not competing places that define it.